Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/GameNetPortal.cs

```diff
@@ -242,8 +242,9 @@ public async void StartUnityRelayHost()
 
         void StartHost()
         {
+            // First initialize the host data so that objects that need it during their initial spawn can use it, using ServerClientId as the host's ClientId
+            SessionManager<SessionPlayerData>.Instance.AddHostData(new SessionPlayerData(NetManager.ServerClientId, PlayerName, m_AvatarRegistry.GetRandomAvatar().Guid.ToNetworkGuid(), 0, true));
```

Comment:
Do we have a guarantee that ServerClientId will be the same before and after StartHost?

Reply:
In the sense --> does the SDK team commit to keeping this behaviour in the future?

Reply:
Currently it's a readonly property. I don't know if the SDK team commit to keep it that way, but I don't see why that would change. I'll ask just to be sure

Reply:
I was told that it was not safe to assume that it would not change, so I'll be implementing a different solution